static/app/components/replays/replayView.tsx

```diff
@@ -25,7 +28,11 @@ function ReplayView({toggleFullscreen}: Props) {
       <Panel>
         <ReplayPlayer />
       </Panel>
-      {hasNewTimeline ? null : <ReplayController toggleFullscreen={toggleFullscreen} />}
+      {isFullscreen ? (
+        <ReplayController toggleFullscreen={toggleFullscreen} />
+      ) : hasNewTimeline ? null : (
+        <ReplayController toggleFullscreen={toggleFullscreen} />
+      )}
```

Comment:
i think you can do something like
```suggestion
 {isFullscreen || !hasNewTimeline ? (
        <ReplayController toggleFullscreen={toggleFullscreen} />
      ) :  null 
 )}
```
to simply

Reply:
thank you!!! I got confused trying to simplify it myself and gave up :(